docs/migration-guide-2.3-2.4.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Migration Guide 2.3-2.4
+---
+
+In Kyma 2.0, we introduced a [new way to reach registered services and some Application Connectivity improvements](https://kyma-project.io/blog/2021/12/7/release-notes-20/#application-connectivity), which were related to [Service Catalog removal](https://kyma-project.io/blog/2021/12/7/release-notes-20/#service-catalog-deprecation-update).
+We then also informed you we'd be removing the support for the old way soon and encouraged you to switch to the new flow making use of Central Application Gateway. In this release, we're following up on this promise and switching off the support for the old flow. 
```

Comment:
```suggestion
We then also informed you we'd be removing the support for the old way soon and encouraged you to switch to the new flow making use of Central Application Gateway. In this release, we're following up on this promise and switching off the support for the old flow.

By default, we will not remove any components from your cluster. If for some reason you would like to keep the Service Catalog-related resources then you can skip this migration guide, and just not run the mentioned script.
```

Reply:
I am not sure how to make sure that the message about the script being "optional" stands out. @majakurcius perhaps you can change/edit that text according to your opinion, I have no preferences, just would like to have that stated clearly. 🙂

Reply:
I changed the word "must" to "should" in `these resources must be deleted` and added your note to the end of the guide (so that it naturally connects to the previous information about the script.